src/components/NewBook.js

```diff
@@ -27,16 +27,16 @@ const AddBookForm = () => {
     setUserInput({
       title: '',
       author: '',
-      genre: '',
+      category: '',
     });
   };
 
   return (
     <div className="form-wrapper">
-      <h1>ADD NEW BOOK</h1>
+      <h3>ADD NEW BOOK</h3>
       <form className="book-form" onSubmit={handleSubmit}>
         <input
-          className="book-input"
+          className="title-input"
           type="text"
           name="title"
           value={userInput.title}
```

Comment:
- [ ] Kindly ensure you style the form component as well to fit the design - noting the fonts and sizes of the text.

  Your Implementation | Zeplin Design
  :---:|:---:
  ![wqp](https://user-images.githubusercontent.com/36542986/218398559-6af6cba8-1ccd-4d3f-982b-9be11e4a9f69.jpg) | ![ewpw](https://user-images.githubusercontent.com/36542986/218398554-eca0d06e-c6b7-43e0-80eb-715cadca171f.jpg)